src/txn/txn.c

```diff
@@ -921,6 +930,12 @@ __wt_txn_rollback(WT_SESSION_IMPL *session, const char *cfg[])
 	txn->mod_count = 0;
 
 	__wt_txn_release(session);
+	/*
+	 * We're between transactions, if we need to block for eviction, it's
+	 * a good time to do so.  Note that we must ignore any error return
+	 * because the user's data is committed.
+	 */
+	(void)__wt_cache_eviction_check(session, false, readonly, NULL);
```

Comment:
We don't actually have to ignore the error in the call from `wt_txn_rollback` and the comment mentions commit, which is incorrect (just copied from `wt_txn_commit`).